task_test.go

```diff
@@ -43,7 +43,9 @@ func (s *TaskSuite) SetupTest() {
 
 	s.req = task.Request{
 		ResponsePath: filepath.Join(s.outputsDir, "response.json"),
-		Config:       task.Config{},
+		Config: task.Config{
+			BuildkitSecrets: make(map[string]string),
```

Comment:
I think it's cleaner to not have to fill this in, since this is suite-wide setup. (1/2)